gulpfile.js

```diff
@@ -21,6 +21,8 @@ var nls = require('vscode-nls-dev');
 
 require('./tasks/packagetasks')
 
+const languages = [ /* for example { folderName: 'ru', id: 'ru' } */ ];
```

Comment:
Should we start with ```nls.coreLanguages```? Seems like that's what used to be passed to ```createAdditionalLanguageFiles``` before this change. Do you know what's in that collection?